lib/phoenix/live_dashboard/live/processes_live.ex

```diff
@@ -93,24 +104,48 @@ defmodule Phoenix.LiveDashboard.ProcessesLive do
   end
 
   defp format_name_or_initial_call(name) when is_atom(name), do: inspect(name)
-  defp format_name_or_initial_call({m, f, a}), do: Exception.format_mfa(m, f, a)
+  defp format_name_or_initial_call(call), do: SystemInfo.format_call(call)
 
   @impl true
   def handle_info({:node_redirect, node}, socket) do
     {:noreply, push_redirect(socket, to: self_path(socket, node, socket.assigns.params))}
   end
 
   def handle_info(:refresh, socket) do
+    if pid = socket.assigns.pid, do: send_update(ProcessInfoComponent, id: pid)
     {:noreply, fetch_processes(socket)}
   end
 
   @impl true
   def handle_event("select_limit", %{"limit" => limit}, socket) do
     params = %{socket.assigns.params | limit: limit}
-    {:noreply, push_patch(socket, to: self_path(socket, socket.assigns.menu.node, params))}
+    {:noreply, push_patch(socket, to: self_path(socket, menu_node(socket), params))}
+  end
+
+  @impl true
+  def handle_event("show_info", %{"pid" => list_pid}, socket) do
+    pid = decode_pid(list_pid)
+    {:noreply, push_patch(socket, to: process_info_path(socket, pid, socket.assigns.params))}
+  end
+
+  defp process_info_path(socket, pid, params) when is_pid(pid) do
+    live_dashboard_path(socket, :processes, node(pid), [encode_pid(pid)], params)
   end
 
   defp self_path(socket, node, params) do
     live_dashboard_path(socket, :processes, node, [], params)
   end
+
+  defp menu_node(socket), do: socket.assigns.menu.node
```

Comment:
Move this function to `Phoenix.LiveDashboard.Web` and re-use it everywhere in the project perhaps?